Index: freebsd-buildutils/src/usr.bin/make/Makefile
===================================================================
--- freebsd-buildutils.orig/src/usr.bin/make/Makefile
+++ freebsd-buildutils/src/usr.bin/make/Makefile
@@ -6,6 +6,7 @@
 
 PROG=	make
 CFLAGS+=-I${.CURDIR}
+LDADD+=-lbsd
 SRCS=	arch.c buf.c cond.c dir.c for.c hash.c hash_tables.c job.c	\
 	lst.c main.c make.c parse.c proc.c shell.c str.c suff.c targ.c	\
 	util.c var.c
Index: freebsd-buildutils/src/usr.sbin/mtree/Makefile
===================================================================
--- freebsd-buildutils.orig/src/usr.sbin/mtree/Makefile
+++ freebsd-buildutils/src/usr.sbin/mtree/Makefile
@@ -9,6 +9,7 @@ PROG=	fmtree
 MAN=	fmtree.8 mtree.5
 SRCS=	compare.c crc.c create.c excludes.c misc.c mtree.c spec.c verify.c
 SRCS+=	specspec.c
+LDADD+=	-lbsd
 
 
 .if ${MK_NMTREE} == "no"
Index: freebsd-buildutils/src/usr.bin/brandelf/Makefile
===================================================================
--- freebsd-buildutils.orig/src/usr.bin/brandelf/Makefile
+++ freebsd-buildutils/src/usr.bin/brandelf/Makefile
@@ -1,5 +1,6 @@
 # $FreeBSD$
 
 PROG=	brandelf
+LDADD=	-lbsd
 
 .include <bsd.prog.mk>
Index: freebsd-buildutils/src/usr.bin/cksum/Makefile
===================================================================
--- freebsd-buildutils.orig/src/usr.bin/cksum/Makefile
+++ freebsd-buildutils/src/usr.bin/cksum/Makefile
@@ -3,6 +3,7 @@
 
 PROG=	cksum
 SRCS=	cksum.c crc.c print.c sum1.c sum2.c crc32.c
+LDADD+=	-lbsd
 SIMLINKS=	${BINDIR}/cksum ${BINDIR}/sum
 MLINKS=	cksum.1 sum.1
 
Index: freebsd-buildutils/src/usr.bin/file2c/Makefile
===================================================================
--- freebsd-buildutils.orig/src/usr.bin/file2c/Makefile
+++ freebsd-buildutils/src/usr.bin/file2c/Makefile
@@ -3,6 +3,7 @@
 .include <bsd.own.mk>
 
 PROG=	file2c
+LDADD+=	-lbsd
 
 .if ${MK_TESTS} != "no"
 SUBDIR+= tests
