From: Helge Deller <deller@gmx.de>
Subject: Do not use -fstack-protector on hppa/parisc arch.
Bug-Debian: https://bugs.debian.org/758377
Forwarded: not-needed

Index: freebsd-buildutils/src/share/mk/bsd.sys.mk
===================================================================
--- freebsd-buildutils.orig/src/share/mk/bsd.sys.mk
+++ freebsd-buildutils/src/share/mk/bsd.sys.mk
@@ -144,11 +144,12 @@ CXXFLAGS+=	 ${CXXFLAGS.gcc}
 .endif # !EARLY_BUILD
 
 .if ${MK_SSP} != "no" && ${MACHINE_CPUARCH} != "ia64" && \
-    ${MACHINE_CPUARCH} != "arm" && ${MACHINE_CPUARCH} != "mips"
+    ${MACHINE_CPUARCH} != "arm" && ${MACHINE_CPUARCH} != "mips" && \
+    ${MACHINE_CPUARCH} != "hppa" && ${MACHINE_CPUARCH} != "alpha"
 # Don't use -Wstack-protector as it breaks world with -Werror.
 SSP_CFLAGS?=	-fstack-protector
 CFLAGS+=	${SSP_CFLAGS}
-.endif # SSP && !IA64 && !ARM && !MIPS
+.endif # SSP && !IA64 && !ARM && !MIPS && !HPPA && !ALPHA
 
 # Allow user-specified additional warning flags
 CFLAGS+=	${CWARNFLAGS}
