Subject: use relative path in manpage symlinks
From: Steven Chamberlain <steven@pyro.eu.org>
Date: Sat, 05 Mar 2016 15:31:00 +0000

See https://bugs.debian.org/749327

--- a/src/share/mk/bsd.man.mk
+++ b/src/share/mk/bsd.man.mk
@@ -211,12 +211,12 @@
 			[123]) echo "warn: empty MLINK: $$1 $$2 $$3"; break;; \
 		esac; \
 		name=$$1; shift; sect=$$1; shift; \
-		l=${DESTDIR}${MANDIR}$${sect}${MANSUBDIR}/$$name; \
+		l=$$name; \
 		name=$$1; shift; sect=$$1; shift; \
 		t=${DESTDIR}${MANDIR}$${sect}${MANSUBDIR}/$$name; \
 		${ECHO} $${t}${ZEXT} -\> $${l}${ZEXT}; \
 		rm -f $${t} $${t}${MCOMPRESS_EXT}; \
-		${INSTALL_LINK} $${l}${ZEXT} $${t}${ZEXT}; \
+		${INSTALL_SYMLINK} $${l}${ZEXT} $${t}${ZEXT}; \
 	done
 .if defined(MANBUILDCAT) && !empty(MANBUILDCAT)
 	@set ${MLINKS:C/\.([^.]*)$/.\1 \1/}; \
